internal/app/singularity/cache_clean_linux.go

```diff
@@ -46,28 +57,28 @@ func cleanBlobCache() error {
 	}
 
 	return nil
-
 }
```

Comment:
Ok, `cleanLibraryCache`, `cleanBlobCache`, `cleanShubCache` and `cleanOciCache` all do the same, they only differ on the param passed on `os.RemoveAll`, which is the cache type. Wouldn't it be better to instead create a single function where it receives the type and it removes the specified cache? I mean something like:

```
 func cleanSpecifiedCache(cacheType string) error{
            switch cacheType {
	        case "library":
                    cacheToRemove:= cache.Library()
                case "blob":
                    cacheToRemove:= cache.OciBlob()
                case "shub":
                    cacheToRemove:= cache.Shub()
                // add any additional cases needed

              sylog.Debugf("Removing: %v", cacheToRemove)

	err := os.RemoveAll(cacheToRemove)
	if err != nil {
		return fmt.Errorf("unable to clean %s cache: %v", cacheType , err)
	}

	return nil
}
```
That would remove functions:  `cleanLibraryCache`, `cleanBlobCache`, `cleanShubCache` and `cleanOciCache`